### ISSUE TITLE
[Story Point Value: 6] {Assignee: Tristan} Implement loss (cost) functions for the neural network for model training. (Loss computation is still not implemented). Implement using binary cross entropy, cross entropy, MSE

### ISSUE THREAD

**Issue by tzwchen** [jwannamaker/neural-web-tool#33]

**Comment by tzwchen** [comment]
Implemented using MSE
